ChangeLog.md

```diff
@@ -1,5 +1,289 @@
 # Changelog for haskell-language-server
 
+## 1.6.0
+
+Time for a new and exciting hls release:
+
+- It includes *three* brand new plugins:
+  - [Alternate number literals](https://haskell-language-server.readthedocs.io/en/latest/features.html#convert-numbers-to-alternative-formats) thanks to @drsooch
+  - [Qualify imported names](https://haskell-language-server.readthedocs.io/en/latest/features.html#qualify-imported-names) thanks to @eddiemundo
+  - New plugin to support [selection range](https://haskell-language-server.readthedocs.io/en/latest/features.html#selection-range) (aka double click text selection) thanks to @kokobd
+- Finally hls [supports *ghc 9.2.1*](https://github.com/haskell/haskell-language-server/issues/2179)
+  - Including core features and [many plugins](https://haskell-language-server.readthedocs.io/en/latest/supported-versions.html#plugins-support-by-ghc-version))
+  - Thanks to a great collective effort coordinated by @pepeiborra and with the help of @wz1000, @mpickering and @alanz among others
+- Hls now also [supports *ghc 9.0.2*](https://github.com/haskell/haskell-language-server/issues/297) with support for all plugins but the stylish-haskell formatter
```

Comment:
As stylish-haskell remains as a "special" case, maybe it is worth noting that the support is blocked by an upstream issue? Or even better, with a link to the issue: https://github.com/haskell/stylish-haskell/issues/378

Reply:
Also,
```suggestion
- Hls now also [supports *ghc 9.0.2*](https://github.com/haskell/haskell-language-server/issues/297) with all plugins but the stylish-haskell formatter
```
the repetition of "support" sounds a bit redundant.